testutil/beaconmock/options.go

```diff
@@ -268,6 +268,39 @@ func WithDeterministicAttesterDuties(factor int) Option {
 	}
 }
 
+// WithDeterministicSyncCommDuties configures the mock to provide deterministic sync committee duties based on provided arguments and config.
+// Note that it depends on ValidatorsFunc being populated, e.g. via WithValidatorSet.
+func WithDeterministicSyncCommDuties() Option {
```

Comment:
remove this, we already have `WithSyncCommitteeDuties`, can remove it to `WithDeterministicSyncCommDuties`

Reply:
oh, right, i see it now. thanks!